packages/twenty-server/src/modules/workflow/workflow-trigger/services/workflow-trigger-service.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
```

Comment:
not sure about adding a module file in a `service` folder, seems to be an antipattern to me. Why not adding thoses services in the `workflow-trigger.module.ts`?